stable/filebeat/templates/service.yaml

```diff
@@ -2,6 +2,10 @@
 kind: Service
 apiVersion: v1
 metadata:
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/port: "9479"
+    prometheus.io/scrape: "true"
```

Comment:
We should likely not have these when the serviceMonitor is disabled - https://github.com/helm/charts/blob/master/stable/filebeat/values.yaml#L150

Reply:
Done